client/src/components/TextAvatar/index.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import { Avatar } from "antd";
+
+import { ROYAL_BLUE, SELAGO } from "constants/colors";
+
+const TextAvatar = styled(Avatar)`
```

Comment:
Nice! This reminds me - the avatars are going to have name initials if there is no photo. I remember researching a third-party library that generates avatars that have initials. We can implement this later.

Reply:
This component receives initials from the user name if the user has no photo. Current posts service doesn't return this field so I need to add initials avatar.